Makefile

```diff
@@ -0,0 +1,31 @@
+# Makefile for generating Go code from .proto files
```

Comment:
It's a general makefile that includes compiling protos. Not a blocker, but should include:
- Building the microservice
- Running the microservice
- Building/pushing image to a container repository (should probably default to github's)